packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/library_indirect/Makefile

```diff
@@ -0,0 +1,27 @@
+LEVEL = ../../../../make
+
+EXE=a.out
+SWIFT_SOURCES=main.swift
+SWIFTFLAGS_EXTRAS= -I$(shell pwd)
```

Comment:
Does `-I. ` also work?

Reply:
Otherwise `Makefile.rules` defines `$(BUILDDIR)` as `$(shell pwd)`.

Reply:
Oops, I forgot this one. I'll get it in the next rebase.